File: 0028-find-the-index-of-the-first-occurrence-in-a-string/0028-find-the-index-of-the-first-occurrence-in-a-string.java
class Solution {
    public int strStr(String haystack, String needle) {
        if(haystack.contains(needle)){
            int i=0;
            while(i<haystack.length()){
                if(haystack.substring(i,needle.length()+i).equals(needle)){
                    return i;
                }
                i++;
            }
        }
        return -1;
    }
}

File: 0028-find-the-index-of-the-first-occurrence-in-a-string/0028-find-the-index-of-the-first-occurrence-in-a-string.js
/**
 * @param {string} haystack
 * @param {string} needle
 * @return {number}
 */
var strStr = function(haystack, needle) {
    if(haystack.includes(needle)){
        let i=0;
        while(i<haystack.length){
            if(haystack.substring(i,i+needle.length) === needle){
                return i;
            }
            i++;
        }
    }
    return -1;
};